controllers/front/ProductController.php

```diff
@@ -1228,6 +1228,16 @@ public function getTemplateVarPage()
         }
         $page['admin_notifications'] = array_merge($page['admin_notifications'], $this->adminNotifications);
 
+        $idProductAttribute = $this->getIdProductAttribute(true);
```

Comment:
can you please create a new function with a meaningful name and don't use a boolean flag?